develop-docs/sdk/telemetry/logs.mdx

```diff
@@ -303,7 +309,67 @@ If a log is generated by an SDK integration, the SDK should also set the `sentry
 }
 ```
 
-Beyond these attributes, we are exploring if the SDK should also send OS, user, and device information automatically (via reading the appropriate contexts from the scope). Given this behaviour can easily be added as a new feature to the SDK, it does not have to be part of the initial SDK implementation until we make a finalized decision.
+#### User Attributes
+
+If `sendDefaultPii`/`send_default_pii` is set to `true` in the SDK, the SDK should attach the following user data if available:
+
+1. `user.id`: The user ID. Maps to `id` in the [User](/sdk/data-model/event-payloads/user/) payload.
+2. `user.name`: The username. Maps to `username` in the [User](/sdk/data-model/event-payloads/user/) payload.
+3. `user.email`: The email address. Maps to `email` in the [User](/sdk/data-model/event-payloads/user/) payload.
+
+```json
+{
+  "user.id": "123",
+  "user.name": "john.doe",
+  "user.email": "john.doe@example.com"
+}
+```
+
+#### User Agent Parsing
+
+By default, relay should parse the user agent attached to an incoming log envelope to parse `browser` and `os` information for logs. These attributes should be attached by relay, but SDKs can attach them if they do not forward a user agent when sending logs to Sentry.
+
+1. `browser.name`: Display name of the browser application. Maps to `name` in the [Contexts](/sdk/data-model/event-payloads/contexts/#browser-context) payload.
+2. `browser.version`: Version string of the browser. Maps to `version` in the [Contexts](/sdk/data-model/event-payloads/contexts/#browser-context) payload.
+
+```json
+{
+  "browser.name": "Chrome",
+  "browser.version": "120.0"
+}
+```
+
+#### Backend SDKs
+
+For backend SDKs (Node.js, Python, PHP, etc.), the SDKs should attach the following:
+
+1. `server.address`: The address of the server that sent the log. Equivalent to `server_name` we attach to errors and transactions.
+
+```json
+{
+  "server.address": "foo.example.com"
+}
+```
+
+#### Mobile, Desktop, and Native SDKs
+
+For mobile, desktop, and native SDKs (Android, iOS, Electron, etc.), the SDKs should attach the following:
```

Comment:
Again why is this list incomplete? What drove us to only add a subset? It's OK, but we should point out why here.

Reply:
I added a note as part of 9fc68094ccf8db1255c3e2c24e22a650b270a14b to address this.